src/equations/hyperbolic_serre_green_naghdi_1d.jl

```diff
@@ -433,7 +433,7 @@ end
 
 # The entropy/energy takes the whole `q` for every point in space
 """
-    energy_total_modified(q_global, equations::HyperbolicSerreGreenNaghdiEquations1D, cache)
+    energy_total_modified!(q_global, equations::HyperbolicSerreGreenNaghdiEquations1D, cache)
```

Comment:
```suggestion
    energy_total_modified!(e, q_global, equations::HyperbolicSerreGreenNaghdiEquations1D, cache)
```

We also need to update the docstring accordingly. We may also consider exporting this function - or should prefix it with `DispersiveShallowWater.`. We may also mention this in the general docstring of `energy_total_modified`.